src/Neo/SmartContract/Native/ContractManagement.cs

```diff
@@ -92,7 +92,19 @@ internal override async ContractTask OnPersist(ApplicationEngine engine)
                         oldContract.Manifest = contractState.Manifest;
                     }
 
-                    await contract.Initialize(engine, hf);
+                    // Initialize native contract for all hardforks
+                    if (hfs?.Length > 0)
+                    {
+                        foreach (var hf in hfs)
+                        {
+                            await contract.Initialize(engine, hf);
```

Comment:
In are in a name violation of the method `Initialize`; why is stuff like this getting merged in repo. We all agree to follow the naming conventions that was proposed.

So Please rename `Initialize` to `InitializeAsyn`

Thanks 😃

Reply:
Agree, but we can rename it in a different pr, there are more methods like this and this pr is for a different topic